src/containers/Tabs/Account.tsx

```diff
@@ -47,101 +48,107 @@ const AccountTabContainer: React.FC<AccountTabContainerProps> = ({ isOpen }) =>
     openToast
   } = useTab(whitelist, (identifier: string) => ({ address: identifier }));
 
-  const handleAdd = async (value: string) => {
-    // const gas = await accountRulesContract!.estimate.addAccount(value)
-    try {
-      const tx = await accountRulesContract!.functions.addAccount(value);
-      toggleModal('add')();
-      addTransaction(value, PENDING_ADDITION);
-      const receipt = await tx.wait(1); // wait on receipt confirmations
-      const addEvent = receipt.events!.filter(e => e.event && e.event === 'AccountAdded').pop();
-      if (!addEvent) {
-        openToast(value, FAIL, `Error while processing account: ${value}`);
-      } else {
-        const addSuccessResult = idx(addEvent, _ => _.args[0]);
-        if (addSuccessResult === undefined) {
+  if (!!accountRulesContract) {
+    const handleAdd = async (value: string) => {
+      // const gas = await accountRulesContract!.estimate.addAccount(value)
```

Comment:
Do we need this comment here/

Reply:
Looks like some typescript WIP or something...  Will confirm.